### ISSUE TITLE
Scheduling for a dataset publishes for another

### ISSUE THREAD

**Issue by frederikgdl** [sanity-io/sanity-plugin-scheduled-publishing#10]
I have a setup with two datasets, `dev` and `prod` and I'm using the `__experimental_spaces` config in `sanity.json` to have them be selectable in the studio. I was testing the new scheduled publishing, and when I try to schedule publishing of a document in the studio for the `dev` dataset the document (with the same ID) in the `prod` dataset was published instead.

There seems to be an issue with
```
import sanityClient from 'part:@sanity/base/client'

console.log(sanityClient.config().dataset)
```
which logs `prod` even if I have changed to `dev` in the dropdown in the studio.

My `sanity.json` contains the following:
```
"api": {
  "projectId": "...",
  "dataset": "prod"
},
"__experimental_spaces": [
  {
    "name": "production",
    "title": "Prod",
    "default": true,
    "api": {
      "projectId": "...",
      "dataset": "prod"
    }
  },
  {
    "name": "dev",
    "title": "Dev",
    "api": {
      "projectId": "...",
      "dataset": "dev"
    }
  }
]
```

Is this not supported, am I doing something wrong, or is there a bug?

**Which versions of Sanity are you using?**
```
@sanity/cli                   2.29.8 (up to date)
@sanity/base                  2.29.8 (up to date)
@sanity/components            2.14.0 (up to date)
@sanity/core                  2.29.5 (up to date)
@sanity/default-layout        2.29.8 (up to date)
@sanity/default-login         2.29.8 (up to date)
@sanity/desk-tool             2.29.8 (up to date)
@sanity/eslint-config-studio   2.0.0 (up to date)
@sanity/production-preview    2.29.3 (up to date)
@sanity/scheduled-publishing   0.1.5 (up to date)
@sanity/vision                2.29.8 (up to date)
```

**What operating system are you using?**
`macOS Monterey Version 12.3.1`

**Which versions of Node.js / npm are you running?**

```
8.5.5
v16.14.2
```

**Comment by robinpyon** [comment]
Thanks for flagging this @frederikgdl. 

You weren't doing anything wrong! I can confirm this issue has since been fixed in `0.1.6`. Please shout if you're still having issues here.

**Comment by frederikgdl** [comment]
Thank you @robinpyon, it's working for me now :D
